docs/theme/js/theme.js

```diff
@@ -162,129 +147,107 @@ function handleNavScroll() {
       if (hasMenuItem(id)) {
         selectMenuItem(id);
         return;
-      }      
+      }
     }
   }
-  
-  function updateMenu() {        
-    for (var i = 0; i < $targets.length; i++) {            
+
+  function updateMenu() {
+    for (var i = 0; i < $targets.length; i++) {
       var id = $targets.eq(i).attr('id');
-      if (checkVisible($targets.eq(i)) && hasMenuItem(id) ) {              
+      if (checkVisible($targets.eq(i)) && hasMenuItem(id) ) {
         selectMenuItem(id);
         return;
-      }             
+      }
     }
 
-    findAndActivateClosest();        
+    findAndActivateClosest();
   }
-  
+
   var hash = window.document.location.hash;
   if (!hash) {
-    updateMenu();  
+    updateMenu();
   } else {
     selectMenuItem(hash.replace('#', ''));
-  } 
-  
-  $container.scroll(debounce(updateMenu, 50));          
-}
-
-// make left menu fixed when scrolling the content
-function handleStickyNav() {    
-  var $content = $(".grv-content");
-  function onScroll(e) {              
-    if ($(".grv-mobile-nav-active").length > 0) {
-      return;
-    }
-
-    if (window.scrollY > 90) {
-      $content.addClass("--fixed-left-nav");
-    } else {
-      $content.removeClass("--fixed-left-nav");
-    }              
   }
-  
-  $(window).scroll(debounce(onScroll, 10));
+
+  $container.scroll(debounce(updateMenu, 50));
 }
 
+
 // creates version selector
 function handleVerSelector() {
   if (!window.grvConfig || !window.grvConfig.docVersions) {
     return;
   }
 
-  var docVersions = window.grvConfig.docVersions || [];
-  var docCurrentVer = window.grvConfig.docCurrentVer;
-    
+  const docVersions = window.grvConfig.docVersions || [];
+  const docCurrentVer = window.grvConfig.docCurrentVer;
+  const viewingLatest = docVersions.length === 0 || docCurrentVer === docVersions[docVersions.length - 1];
+  const $content = $("#docs-content");
+  const $versionContainer = $content.find('h1').first();
+
   function getVerUrl(ver, isLatest) {
     // looks for version number and replaces it with new value
     // ex: http://host/docs/ver/1.2/review -> http://host/docs/ver/4.0
-    var reg = new RegExp("\/ver\/([0-9|\.]+(?=\/.))");
-    var url = window.location.href.replace(reg, '');    
-    var newPrefix = isLatest ? "" : "/ver/" + ver +"/";
-    return url.replace(mkdocs_page_url, newPrefix);    
+    const reg = new RegExp("\/ver\/([0-9|\.]+(?=\/.))");
```

Comment:
since we do not transpile the docs JS, lets not use the modern features here like `const` because it wont work in older browsers. (i.e. in Safari < Sep. 2016)